src/styles/main.scss

```diff
@@ -1,3 +1,56 @@
-body {
-  margin: 0;
+@import "./variables.scss";
+
+.month {
+
+  display: flex;
```

Comment:
```suggestion
.month {
  display: flex;
```

and the checklist 
<img width="730" alt="image" src="https://user-images.githubusercontent.com/102411130/201864488-0169a9d1-e563-4a1e-a8db-7540289f52a7.png">